src/libs/password-ga-kata.js

```diff
@@ -1,31 +1,7 @@
-const KATA_BENDA = [
-  'hotel',
-  'rumah',
-  'makam',
-  'botol',
-  'pisau',
-  'kerbau',
-  'meja',
-  'telepon',
-  'motor',
-  'gitar',
-  'pintu',
-  'kasur'
-];
-const KATA_SIFAT = [
-  'panas',
-  'dingin',
-  'hebat',
-  'bersih',
-  'sehat',
-  'mantap',
-  'jagoan',
-  'sakti',
-  'gila',
-  'nikmat',
-  'lemah',
-  'kuat'
-];
+import WORDS_JSON from '../words.json';
+
+const KATA_BENDA = WORDS_JSON.nouns;
```

Comment:
thanks

Reply:
maybe next time we could tidying up this one

Reply:
Sure! Is it about the character casing of `nouns` or is it about something else?